game/scripts/npc/npc_items_custom.txt

```diff
@@ -1,7 +1,7 @@
-// reflex cores
+// reflex core and reflex core items
 #base "items/item_reflex_core.txt"
 #base "items/reflex_cores/preemptive/item_preemptive.txt"
-
+#base "items/reflex_cores/preemptive/item_preemptive_2b.txt"
```

Comment:
leave the newline between reflex and non-reflex items. It's nice to have some sort of grouping for quick skimming